README.md

```diff
@@ -194,17 +192,73 @@ crafts = [
 _The first craft requires a stone axe to craft the item referenced "1", the second craft enable us to cook a cooked_porkchop into a coal, 
 and the third one is used to craft the item referenced "2" with three sticks aligned in a vertical centered line (notice the whitespaces before and after the "a")_
 
+### Affix
+
+Affix are modifiers who can be applied to item. They are pre-registered in pools (not items pools :wink:).
```

Comment:
`applied to items` ou `applied to an item`. En general en anglais tu mets un determinant au singulier et pas au pluriel, mais tu mixes pas trop les deux  

Ta reference sur les item pools va parler aux connaisseurs, mais je crains quand meme que ca puisse etre confusant d'un regard exterieur (ou si c'est important de le preciser, peut etre etre plus clair `different from the pools used for items`)